packages/leancode_lint/test/lints_test_app/pubspec.lock

```diff
@@ -257,18 +257,18 @@ packages:
     dependency: transitive
     description:
       name: material_color_utilities
-      sha256: f7142bb1154231d7ea5f96bc7bde4bda2a0945d2806bb11670e30b850d56bdec
+      sha256: "0e0a020085b65b6083975e499759762399b4475f766c21668c4ecca34ea74e5a"
       url: "https://pub.dev"
     source: hosted
-    version: "0.11.1"
+    version: "0.8.0"
```

Comment:
Downgrade 🤨

Reply:
Unclear to me why that is. It tells me for example meta is pinned by flutter at 1.11.0, so no clue how come it was 1.12.0 before

Reply:
It might be that the example app doesn't have a specific Flutter version specified, and you ran it with an older version :thinking:

Reply:
![image](https://github.com/leancodepl/flutter_corelibrary/assets/29288116/eabc9d3a-5ff8-412a-bd94-373f096269b6)
 🤷

Reply:
Let's keep it downgraded, then :thinking: